composer.json

```diff
@@ -12,6 +12,10 @@
     "config": {
         "platform": {
             "php": "8.0.99"
+        },
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": true,
+            "infection/extension-installer": false
```

Comment:
Should still not be necessary :thinking:

Reply:
```
$ composer install
dealerdirect/phpcodesniffer-composer-installer contains a Composer plugin which is currently not in your allow-plugins config. See https://getcomposer.org/allow-plugins
Do you trust "dealerdirect/phpcodesniffer-composer-installer" to execute code and wish to enable it now? (writes "allow-plugins" to composer.json) [y,n,d,?] y
infection/extension-installer contains a Composer plugin which is currently not in your allow-plugins config. See https://getcomposer.org/allow-plugins
Do you trust "infection/extension-installer" to execute code and wish to enable it now? (writes "allow-plugins" to composer.json) [y,n,d,?] n
```

🤷‍♂️

Reply:
I can only say `y` or `n` to get rid of the question everytime.

Reply:
Can we add a `--no-interaction` or such, perhaps?

Reply:
```
For additional security you should declare the allow-plugins config with a list of packages names that are allowed to run code. See https://getcomposer.org/allow-plugins
You have until July 2022 to add the setting. Composer will then switch the default behavior to disallow all plugins.
```

Btw `--no-interaction` is in all workflows. I just want to get rid of it on my localhost :)

Reply:
Eh, for localhost you'll have to suffer this pain :D 

Alternatively, we start isolating all this stuff in `Makefile` definitions :thinking:

Reply:
I'm happy Windows user so I'm not sure what will cause me more suffering 😄

Reply:
I'll be honest: that's you looking for trouble :P